sfxclient/httpsink.go

```diff
@@ -243,3 +256,130 @@ func (h *HTTPDatapointSink) encodePostBodyProtobufV2(datapoints []*datapoint.Dat
 	}
 	return body, nil
 }
+
+// HTTPEventSink will accept signalfx events and forward them to SignalFx via HTTP.
+type HTTPEventSink struct {
```

Comment:
while i think individual sinks are maybe good, i think ability to easily get a sink that is both a DSink and an ESink is a good idea. Seems like a client would most likely want to have one that accepts both. 

func NewHTTPSink() would return a composition of the two so the client doesn't have to do it themselves?